test/unit/item.test.js

```diff
@@ -466,6 +466,15 @@ describe('Item', function () {
             item.unsetProtocolProfileBehavior('keyName');
             expect(item.toJSON()).to.have.property('protocolProfileBehavior').that.is.empty;
         });
+
+        it('should not unset protocolProfileBehavior from an Item incase key is not of type string', function () {
+            var item = new Item({
+                protocolProfileBehavior: { keyName: 'null' }
```

Comment:
https://github.com/postmanlabs/postman-collection/pull/1084#discussion_r479958392

I meant to set the `key` to null not set the value as null.